src/mixins/mediaQuery.js

```diff
@@ -0,0 +1,21 @@
+export default {
+  data: () => ({
```

Comment:
Utilize Plugin Platform `this.$f.platform` para pegar o device e o plugin Screen para pegar o tamanhos.

- Platform `this.$f.platform`
```json
{
  "has": {
    "touch": false,
    "webStorage": true
  },
  "within": {
    "iframe": true
  },
  "is": {
    "chrome": true,
    "version": "80.0.3987.149",
    "versionNumber": 80,
    "mac": true,
    "desktop": true,
    "webkit": true,
    "name": "chrome",
    "platform": "mac"
  }
}
```

- Screen `this.$f.screen`

```json
{
  "width": 1510,
  "height": 314,
  "sizes": {
    "sm": 600,
    "md": 1024,
    "lg": 1440,
    "xl": 1920
  },
  "lt": {
    "sm": false,
    "md": false,
    "lg": false,
    "xl": true
  },
  "gt": {
    "xs": true,
    "sm": true,
    "md": true,
    "lg": false
  },
  "xs": false,
  "sm": false,
  "md": false,
  "lg": true,
  "xl": false
}
```